src/components/SettingComponent.js

```diff
@@ -95,7 +95,7 @@ Button.defaultProps = {
   className: null,
 }
 
-export const PopoverColorPicker = ( { name, value, icon, storageKey, onChange, ...props } ) => {
+export const PopoverColorPicker = ( { name, value, icon, storageKey, onChange, disableAlpha, ...props } ) => {
```

Comment:
You don't actually need to specify the `disableAlpha` prop here if you're just passing it through as-is in L140, since `{...props}` will spread it for you.